pages/in-vitro-systems/[id].js

```diff
@@ -12,6 +12,7 @@ import {
   DataItemValue,
   DataPanel,
 } from "../../components/data-area";
+import BiomarkerTable from "../../components/biomarker-table";
```

Comment:
Could you move this above Breadcrumbs to keep them sorted? This also applies to similar changes to other file under pages/.